tests/labeler/chexpert/label.py

```diff
@@ -0,0 +1,230 @@
+import csv
+import os
+import time
+import numpy as np
+import pandas as pd
+from collections import defaultdict
+from knodle.labeler.CheXpert.label import CheXpertLabeler
+from knodle.labeler.CheXpert.preprocessing import Preprocessor
+from knodle.labeler.CheXpert.matching import Matcher
+from knodle.labeler.CheXpert.neg_unc_detection import NegUncDetector
+from knodle.labeler.CheXpert.updating import Updater
+from knodle.labeler.CheXpert.utils import z_matrix_fct, t_matrix_fct, get_rule_idx
+from tests.labeler.chexpert.config_tests import WeatherTestsConfig
+
+
+CHEXPERT_DATA_DIR = os.path.join(os.getcwd())
+
+# RULE DIRECTORIES -----------------------------------------------------------------------------------
+MENTION_DATA_DIR = os.path.join(CHEXPERT_DATA_DIR, "phrases", "mention")
+os.makedirs(MENTION_DATA_DIR, exist_ok=True)
+with open(os.path.join(MENTION_DATA_DIR, 'storm.txt'), 'w') as f:
+    f.write('storm')
+    f.close()
+with open(os.path.join(MENTION_DATA_DIR, 'clouds.txt'), 'w') as f:
+    f.write('cloud')
+    f.close()
+
+UNMENTION_DATA_DIR = os.path.join(CHEXPERT_DATA_DIR, "phrases", "unmention")
+os.makedirs(UNMENTION_DATA_DIR, exist_ok=True)
+
+# PATTERN DIRECTORY ----------------------------------------------------------------------------------
+PATTERNS_DIR = os.path.join(CHEXPERT_DATA_DIR, "patterns")
```

Comment:
I don't think it's good style to have global variables while running tests (in don't like it in general, but a lot of people do it for script files. I'd prefer __main__ then, but that's up for discussion)
Better use fixtures to provide data. See: https://github.com/knodle/knodle/blob/develop/tests/transformation/test_majority.py#L60